fileStore/commands.py

```diff
@@ -8,7 +8,7 @@
 #TODO: Add documentation
 
 
-def save_file_base(spec, file_path, custom=None):
+def save_file_base(spec, file_path, custom=None, collection_version=0):
```

Comment:
How are the internals going to figure out what version of collection it is? That requires additional logic and collections that a fields of a document are stored. That creates significant overhead with increasing number of versions. Also, if such feature is necessary, it must be part of dataBroker. Since this api will be facing dataBroker (and opyhd on collection side), it will be hidden from the user regardless.